src/app/cart/cart.component.js

```diff
@@ -60,8 +60,17 @@ class CartController {
     this.$window.location.href = this.sessionService.getRole() === 'REGISTERED' ? 'checkout.html' : 'sign-in.html';
   }
 
-  donationStartDate( month, day ) {
-    return new Date( 0, Number( month ) - 1, Number( day ) );
+  donationStartDate( config ) {
```

Comment:
@OzzieOrca @Omicron7 Before I write tests, I wanted to get your feedback on this function.  It's getting the start day for each donation in the cart.  Do think this I should do this as part of the "loadCart" function instead?  I'm open to suggestions.